packages/ui-kit/src/components/TimeRangePicker/DateTimeField/DateTimeField.tsx

```diff
@@ -0,0 +1,216 @@
+import {
+  addDays,
+  addMinutes,
+  format as dateFormat,
+  isAfter,
+  isBefore,
+  isValid,
+  parse,
+  subDays,
+  subMinutes
+} from 'date-fns'
+import React, { Dispatch, SetStateAction } from 'react'
+import { DateRange } from 'react-day-picker'
+import { FormField } from '../../FormField'
+import { Input } from '../../Input'
+import componentStyles from './DateTimeField.module.scss'
+
+export type DateTimeFieldProps = {
+  dateRange?: DateRange | null
+  rangeRole: 'from' | 'to'
+  onChange?: Dispatch<SetStateAction<DateRange | null>>
+}
+
+const DATE_FORMAT = 'yyyy/MM/dd'
```

Comment:
I think we should not hardcode this, but instead use the `toLocaleString` functionality. For example,

```ts
new Date().toLocaleString('en-US', { dateStyle: 'short' })
// 4/26/24

new Date().toLocaleString('de-DE', { dateStyle: 'short' })
// 26.04.24
```

For the first parameter, seems we should get it this way (maybe there is a better way?):

```ts
(new Intl.NumberFormat()).resolvedOptions().locale
// en-DE
```